### PR TITLE
updated check-vpc-vpn check to use sensu-cli.  This gives us support …

### PR DESCRIPTION
**Pull request by johntdyer**
Created a new VPC VPN check this time using Ruby and Sensu-Cli in order to support refresh.

### CODE REVIEW

File: bin/check-vpc-vpn.rb [hunk 1]
```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env ruby
+require 'rubygems' if RUBY_VERSION < '1.9.0'
```
Comment (eheydrick) [inline bin/check-vpc-vpn.rb]: We don't support anything older than Ruby 2 so this isn't needed.

File: bin/check-vpc-vpn.rb [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+# DESCRIPTION:
+#   This plugin checks VPC VPN connections to ensure they are up
+#
+# PLATFORMS:
+#   all
+#
+# DEPENDENCIES:
+#   sensu-plugin >= 1.5 Ruby gem
+#   aws-sdk Ruby gem
+#
```
Comment (eheydrick) [inline bin/check-vpc-vpn.rb]: can you update the header to match the standard one at http://sensu-plugins.io/docs/developer_guidelines.html#coding-style and include usage.

**Comment by eheydrick** [comment]
Thanks @johntdyer! Can you also add this new plugin to the README.

**Comment by johntdyer** [comment]
Updated readme and fixes.  Please let me know if you guys need anything else.

File: bin/check-vpc-vpn.rb [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  def run
+    data = fetch_connection_data
+    msg = data[:down_connection_status].join(' | ')
+    name = data[:connection_name]
+    case data[:down_count]
+    when 2 then critical "'#{name}' shows both tunnels as DOWN - [ #{msg} ]"
+    when 1 then warning "'#{name}' shows 1 of 2 tunnels as DOWN - [ #{msg} ]"
+    when 0 then ok "'#{name}' shows 2 of 2 tunnels as UP"
+    else
+      Not sure this could ever happen
```
Comment (eheydrick) [inline bin/check-vpc-vpn.rb]: Mean to have a comment here?

Comment (johntdyer) [inline bin/check-vpc-vpn.rb]: WTH, how did that happen :facepalm:.  Give me a few and I'll fix it

**Comment by johntdyer** [comment]
Added comment from mobile, I think it's right now...

**Comment by eheydrick** [comment]
Looks good to me.

**Pull request merged**